doc/usr/expl/history.md

```diff
@@ -0,0 +1,40 @@
+# History
+
+This project was initially developed in 2016 at a French startup company named
+Stupeflix. It took inspiration from the scene graph paradigm of [OSG] for which
+many clumsy in-house patches were maintained, essentially for video playback
+support and lazy resources management.
+
+The video playback logic was the first foundation to be extracted from these
+patches, and landed in a dedicated module named [sxplayer] (standing for
+"Stupeflix Player"), a free (LGPL) video playback library on top of FFmpeg.
+
+While it helps with maintenance, most of the OSG features remained unneeded,
+the Python bindings were painful, and the complexity of the stack of patches
+added on top was awkward to manage. It was decided to prototype a new engine,
+more oriented toward video compositing rather than video games. Also, mobile
+platforms (iOS and Android) became first class citizen because mobiles
+applications were the main business target.
+
+The "node.gl" name was selected, merely as a joke and word play with "node.js"
+and OpenGL, along with the fact that the scene graph is composed of "nodes".
+
+The project was open-sourced under Apache license when the [GoPro] company
+acquired Stupeflix (late 2016). The main authors of the project, Clément Bœsch
+and Matthieu Bouron, carried on with its development for years, making it a core
+element of the GoPro Quik mobile editing application.
+
+Along the years, the project matured into a significant engine for video
+compositing, animations, video playback, color management, and was notably
+entirely redesigned to support more modern graphics backends.
```

Comment:
redesigned -> improved ? It's still a WIP (sorry).

Reply:
I'm essentially referring to the fact that for years we've been extracting the GL bits out of the nodes and work on introducing new abstracts levels. I think that's accurate to say that we've entirely redesigned the project around this. This being 100% complete or not doesn't really matter.